scripts/variantstore/wdl/extract/create_vat_inputs.py

```diff
@@ -14,7 +14,7 @@ def parse_ancestry_file(ancestry_file):
     {"ERS4367795":"eur","ERS4367796":"eas","ERS4367797":"eur","ERS4367798":"afr","ERS4367799":"oth", ... }
 
     """
-    start = datetime.now()
+    start = datetime.now() # TODO I don think this is where we want this? The goal here was to see how long the whole VAT pipeline would take
```

Comment:
I think we can clean all of this up? if we are running this in a notebook terminal we likely wont see it anyway